apiv6/README.md

```diff
@@ -1,7 +1,14 @@
 # community_sdk_golang - API v6
 
-## Development state
+API v6 client is generated from API specification available at:
+- <https://github.com/kentik/api-schema-public/tree/master/gen/openapiv2/kentik/cloud_export/v202101beta1>
+- <https://github.com/kentik/api-schema-public/tree/master/gen/openapiv2/kentik/synthetics/v202101beta1>
 
-Implemented API resources:
-- cloud export
-- synthetics
\ No newline at end of file
+## Generate client from specification
+
+Strictly saying, the code is generated from OpenAPI 3.0.0 specification.  
+To convert Swagger 2.0 -> OpenAPI 3.0.0, one can use the following converter: <https://mermade.org.uk/openapi-converter>  
```

Comment:
Ideally we could have a script that downloads the latest openapi 2.0, converts to openapis 3.0 and generates the code/documents. What do you think about putting such task to backlog?

Reply:
We can consider such an improvement if it feels worthwhile

Reply:
I like the idea. Any automation that removes repeated mechanical work is helpful. I'll also discuss with engineering options for publishing OpenAPI 3.0.0 compatible spec to avoid the need for conversion.

Reply:
OpenAPI 3.0.0 is the most recent and covers the Swagger spec functionality. Maybe it's a good idea to turn to OpenAPI 3.0.0 spec altogether. Just thinking